api/src/pcapi/core/mails/transactional/pro/invoice_available_to_pro.py

```diff
@@ -1,15 +1,19 @@
 from pcapi.core import mails
+from pcapi.core.finance.api import get_invoice_period
```

Comment:
(style) On préfère généralement importer les modules plutôt que les fonctions seules.

```suggestion
import pcapi.core.finance.api as finance_api
```